src/components/menu/styles.ts

```diff
@@ -0,0 +1,68 @@
+import { styled } from '@linaria/react'
+import { Text2XS } from '../typography'
+
+export const ElMenuPopover = styled.div`
+  position: absolute;
+  white-space: nowrap;
+  z-index: 98;
+`
+
+export const ElMenu = styled.div`
+  overflow: hidden;
+  display: flex;
+  min-width: 200px;
+  width: fit-content;
+  padding: var(--spacing-2) var(--spacing-none);
+  border-radius: var(--corner-default);
+  background-color: var(--fill-white);
+  box-shadow: 0px 4px 16px 0px #222b3329;
+`
```

Comment:
**question:** Does the Design System place the `box-shadow` on the MenuList component? It's not clear to me if this should belong to the Popover or the MenuList 🤔

Reply:
>Does the Design System place the box-shadow on the MenuList component?

yes, it has a box-shadow, but for where I should put it, in my understanding, `MenuList` is a styled container of the `MenuGroup`, and the `Popover` is just container which contain the logic.

question: should we combine them instead ?

Reply:
> in my understanding, MenuList is a styled container of the MenuGroup, and the Popover is just container which contain the logic.

Yeah, I think that's the way I lean too, so let's leave it like that and see what we learn 👍